recipe/meta.yaml

```diff
@@ -46,11 +46,17 @@ test:
 
 about:
   home: http://hdfscli.readthedocs.org
```

Comment:
To update:
doc_url: https://hdfscli.readthedocs.io/
dev_url: https://github.com/mtth/hdfs/
license_family: MIT

Reply:
Thanks! Made those changes